web/static/assets/audio/playlists.json

```diff
@@ -0,0 +1,55 @@
+{
+    "effects": {
+        "eat": [
+            { "url": "./Cartoon Munch Sound Effect.mp3", "volume": 0.2 }
+            ,{ "url": "./Cartoon Bite Sound Effect.mp3", "volume": 0.2 }
+        ]
+        ,"explode": [
+	    { "url": "./Barrel Exploding-SoundBible.com-1134967902.mp3" }
+	    ,{ "url": "./Grenade-SoundBible.com-2124844747.mp3" }
+	    ,{ "url": "./Blast-SoundBible.com-2068539061.mp3" }
+	    ,{ "url": "./Bomb 3-SoundBible.com-1260663209.mp3" }
+	    ,{ "url": "./Explosion-SoundBible.com-2019248186.mp3" }
+	    ,{ "url": "./Grenade Explosion-SoundBible.com-2100581469.mp3" }
+            ,{ "url": "./Wilhelm_Scream.ogg" }
+        ]
+    }
+    ,"music": {
+	"prologue": [
+	    {"url": "http://drive.google.com/uc?export=download&id=0B0Y-FESP0bc-ZkdYblAwMVpMMEU"
+             ,"name": "14 - Vena's Dance.mp3" }
+	    ,{"url": "http://drive.google.com/uc?export=download&id=0B0Y-FESP0bc-Nm8zR3d6TWZCUkk"
+              ,"name": "9-42 The Apple_ M4 tk 1.mp3" }
+	    ,{"url": "http://drive.google.com/uc?export=download&id=0B0Y-FESP0bc-bGUzSU8wZ19sMzQ"
+              ,"name": "9-39 The Apple_ M1 tk 1.mp3" }
+	]
+	,"fight": [
+	    { "url": "http://drive.google.com/uc?export=download&id=0B0Y-FESP0bc-UHpzSzNkOXBUMEU"
+               ,"name": "Speedy Reader" }
+	    ,{ "url": "http://drive.google.com/uc?export=download&id=0B0Y-FESP0bc-VWdZT1BwZTU1WVk"
+               ,"name": "Temporal Wake"}
+	    ,{ "url": "http://drive.google.com/uc?export=download&id=0B0Y-FESP0bc-ZFdHZEpDYzFDd2s"
+               ,"name": "Surprise Attack" }
+	    ,{ "url": "http://drive.google.com/uc?export=download&id=0B0Y-FESP0bc-UUszZWRuZzZ3NGs"
+               ,"name": "Klingon Battle" }
+	    ,{ "url": "http://drive.google.com/uc?export=download&id=0B0Y-FESP0bc-alhGWW1RMWJmSk0"
+               ,"name": "7-12 Amok Time_ Ritual _ To the Deat" }
+	    ,{ "url": "http://drive.google.com/uc?export=download&id=0B0Y-FESP0bc-YVlfaTc2RkRPTGc"
+               ,"name": "Amok Time The Ancient Combat" }
+            ,{ "url": "http://drive.google.com/uc?export=download&id=0B0Y-FESP0bc-Nkd5RnB5OG5KY2s"
+               ,"name": "James Horner - 1986 - Aliens - Deluxe Edition - Ripley's Rescue" }
+            ,{ "url": "http://drive.google.com/uc?export=download&id=0B0Y-FESP0bc-dHJ3dk1HaFB1cVE"
+               ,"name": "Utah Saints - Utah Saints Take On The Theme From Mortal Kombat" }
+            ,{ "url": "http://drive.google.com/uc?export=download&id=0B0Y-FESP0bc-YlI4YWM3WHMzcXM"
+               ,"name": "Traci Lords - Control (Juno Reactor Instrumental)"}
```

Comment:
actually, you can!  I had a long chat with [SOCAN](https://www.socan.ca/licensees/faq-licensing)  today, and playing music like this falls under [category 8](https://www.socan.ca/tariff/Infocard_8_EN.pdf), and that's paid by the hall.  The conference center is responsible for paying the fee, which they do quarterly.  I also asked about playing music served up over the internet, and we're good for that too.  Strange feeling being legal!  Who knew all you had to do was ask.